src/web/app/pages-help/instructor-help-page/instructor-help-page.component.scss

```diff
@@ -19,11 +19,15 @@
   padding-bottom: 5px;
 }
 
-@media (max-width: 480px) {
+@media (max-width: 1500px) {
```

Comment:
Probably we can reduce some `max-width` here so that sidebar does not disappear too early. Maybe try out somewhere 600-800px.

Reply:
Sure. Lemme try. I chose this number because having a lower number makes the sidebar appear over the tables.